make.sh

```diff
@@ -83,8 +83,11 @@ fi
 
 cd ios
 
+echo "=> Removing unnecessary files and folders"
```

Comment:
How about the `Domain` layer? 🤔

Reply:
There might be a platform-specific logic, and we're not sharing the ViewModel in KMM due to its limitation. So I followed most of the KMM-IC project's structure and kept the Domain.

Reply:
ViewModel is related to Presentation Layer, Domain layer code should be shared in the shared module (KMM module), so I would prefer to remove the domain folder

Reply:
However, I think we still need to the domain layer to wrap UseCase, so I think we still need to keep it